### ISSUE TITLE
ofToggleFullscreen changes screen width / height after four calls - OF 0.10.0 (OSX)

### ISSUE THREAD

**Issue by dimitre** [openframeworks/openFrameworks#6074]
It goes to fullscreen and back without issues, but the next fullscreen / back to normal window will keep window position and change window size.

Here is the minimum code to reproduce. 
```c++

//--------------------------------------------------------------
void ofApp::keyPressed(int key){
	if (key == 'a') {
		cout << "toggle" << ofRandom(0,999) << endl;
		ofToggleFullscreen();
	}
}

```

try testing using a small window
```c++
	ofSetupOpenGL(300,300,OF_WINDOW);			// <-------- setup the GL context
```

**Comment by danomatika** [comment]
It seems windowW and windowH are being set to the fullscreen size *before* the window does fullscreen for the second time. The fix is #6092.

**Comment by danomatika** [comment]
_Ignore the various missing commits references above. They were made to the wrong branches before I realized I need to base off of the patch-release branch._
